### PR TITLE
[Fix] Add existing repo_config to output_repos

### PR DESCRIPTION
**Pull request by KevinHock**
Aims to fix https://github.com/pre-commit/pre-commit/issues/660

### CODE REVIEW

File: pre_commit/commands/autoupdate.py [hunk 1]
```diff
@@ -116,8 +116,9 @@ def autoupdate(runner, tags_only, repo=None):
     input_config = load_config(runner.config_file_path)
 
     for repo_config in input_config['repos']:
-        # Skip any repo_configs that aren't the specified repo
+        # Skip updating any repo_configs that aren't for the specified repo
         if repo and repo != repo_config['repo']:
+            output_repos.append(repo_config)
             continue
 
         if is_local_repo(repo_config) or is_meta_repo(repo_config):
```
Comment (asottile) [inline pre_commit/commands/autoupdate.py]: let's combine it with this block :)

Comment (KevinHock) [reply pre_commit/commands/autoupdate.py]: as in the following?

```python
        if is_local_repo(repo_config) or is_meta_repo(repo_config) or (repo and repo != repo_config['repo']):
            output_repos.append(repo_config)
            continue
```

Comment (asottile) [reply pre_commit/commands/autoupdate.py]: yeah, I'd format it like this:

```python
if (
        is_local_repo(repo_config) or
        is_meta_repo(repo_config) or
        (repo and repo != repo_config['repo']
):
```

Comment (KevinHock) [reply pre_commit/commands/autoupdate.py]: Okay, will do.

**Comment by KevinHock** [comment]
Sure thing. (re: regression test)

**Comment by asottile** [comment]
this was released in 1.4.1

**Pull request merged**